### DISCUSSION TITLE
Date picker and checkboxes in filters

### DISCUSSION THREAD

**Discussion by dhruva81** [rappasoft/laravel-livewire-tables#277] [Q&A]
Hi 

I have read complete documentation, but I am not able to find out 

1) how to show date picker and time picker in filters 
2) how to filter between two dates.
3) how to use checkboxes in filters 
4) how to filters using multiple values for one filter type.

Is there any demo available (tailwind), as this package is very powerful and have many advanced features.
As a new laravel learner,  I am not able to understand some concepts from documentation. A demo app will be very helpful for new learner like me :)

Thank you for this amazing package.

**Comment by rappasoft** [comment]
[The filters documentation](https://github.com/rappasoft/laravel-livewire-tables/wiki/Creating-filters) states that the select box is the only type of build-in filter. However, you are free to use a [custom filter view ](https://github.com/rappasoft/laravel-livewire-tables/wiki/Custom-filter-view) and define them however you want and then just write the queries to support them. Will try to make more filter types in the future.

**Reply by dhruva81** [reply]
Thank you for reply.

I have tried custom filter view and able to get my custom filter view (the date picker and checkboxes).

I am not able to filter on changing any input value of custom filter view.

Please guide me, how can I wire:model them.
Kindly help me with any example snippet code.

Thank you

**Reply by rappasoft** [reply]
You need to add the filtering to your query like the normal select box. I haven't tested it with checkboxes. You can see how to do the wire model here: https://github.com/rappasoft/laravel-livewire-tables/blob/master/resources/views/tailwind/includes/filters.blade.php#L53

**Reply by rappasoft** [reply]
The array key of your filter from the filters method with automatically be available to the filters array property on the component.
